lib/money/money.rb

```diff
@@ -272,13 +275,12 @@ def self.disallow_currency_conversion!
   # @return [Money]
   #
   # @see #initialize
-  def self.from_amount(amount, currency = default_currency, bank = default_bank)
+  def self.from_amount(amount, currency = default_currency, *other)
```

Comment:
I don't think it is a good idea to use a wildcard args collection in the declaration of a public API, when the positional values will have meaning.
I do think it would be a good idea to move to an options hash for everything beyond `amount` or `money` (possibly `bank`, if you want to preserve compatibility).
But if you are sticking with positional arguments, spend the extra keystrokes to make the positional argument names explicit.

Reply:
OK, let's make it explicit

Reply:
I think we could still move to an options hash for everything after `amount` and `money` and preserve backwards compatibility by checking if the 3rd param is a Hash.